src/Http/Controllers/Auth/LoginController.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Http\Controllers\Auth;
+
+use Blumilk\Meetup\Core\Http\Controllers\Controller;
+use Blumilk\Meetup\Core\Http\Requests\LoginUserRequest;
+use Blumilk\Meetup\Core\Models\User;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Hash;
+
+class LoginController extends Controller
+{
+    public function formPage()
+    {
+        return view("user.login");
+    }
+    public function login(LoginUserRequest $request)
+    {
+        $validated = $request->validated();
+        $user = User::where("email", $validated["email"])->first();
+        if (!$user || !Hash::check($validated["password"], $user["password"])) {
+            return response("Bad credential", 404);
+        }
+
+        $token = $user->createToken("AccessToken")->plainTextToken;
+        $response = [
+            "user" => $user["email"],
+            "auth_token" => $token,
+        ];
+
+        return response($response, 201);
```

Comment:
After login I would prefer return the view - maybe simple dashboard (you don't need to build an advanced page, maybe something like "Hello in meetup app" in the middle of the page.).